packages/react/src/components/Calendar/Calendar.base.tsx

```diff
@@ -326,6 +327,7 @@ export const CalendarBase: React.FunctionComponent<ICalendarProps> = React.forwa
 
     return (
       <div
+        id={id}
```

Comment:
I think v7 CalendarBase never actually applied id to the div.  The forwarded ref has the HTMLDivElement props, but not the ICalendarProps.  I think we could add id to v8, but then ODSP would see an output change when moving to v8.

Reply:
I checked it before, and the ID does get passed through. Here's a demo

https://codepen.io/micahgodbolt/pen/PoeRpVo?editors=0111